articles/contributing/reporting-bugs.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Reporting bugs | Microsoft Docs
+description: Reporting bugs
+author: cgranade
+ms.author: chgranad
+ms.date: 10/12/2018
+ms.topic: article
+uid: microsoft.quantum.contributing.reporting
+---
+
+# Reporting Bugs #
+
+If you think you've found a bug in a component of the Quantum Development Kit, we would very much appreciate a report.
+After all, someone else may be struggling with the same issue as well; letting us know helps us to get to fixing it for everyone.
+The first step in reporting a bug is to start by looking through the existing issues found on each repository, as it's very possible that someone else has reported the same problem.
+
+If the issue has already been found before, it may be helpful to provide additional information or context that can help us to diagnose and solve the problem.
+In that case, please feel free to participate in discussions on the issue.
+
+If your bug hasn't been reported before, then we'd appreciate if you open a new issue on the repository for the component in question.
+When creating a new issue, you'll be prompted with a template that suggests some information that is often helpful in making informative bug reports:
+
+- Steps to reproduce the problem
+- What you expected to happen
+- What actually happened
+- Information about your software environment (e.g.: OS, .NET Core, and Quantum Development Kit versions)
+
+You can also [create a pull request](https://help.github.com/articles/about-pull-requests/) to fix the bug directly, if it's very straightforward and is not worth the discussion (for example, a typo).
```

Comment:
Better link to our page on pull requests (which in turn should have link to this page)?